src/components/Card.js

```diff
@@ -27,7 +27,7 @@ export default class Card {
 
```

Comment:
`import Popup from "./Popup.js";`


Please, delete imports from `Card.js` 

From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.


But `Popup` is not needed in `Card.js` because you can't create instances from it. It's only for **extending**